<launch>
	<!-- global rosparam -->
	<param name="use_sim_time" value="true"/>

	<!-- param -->
	<param name="weights_path" type="str" value="$(find dnn_attitude_estimation)/weights/camera_regression.pth"/>
	<!-- MAIN -->
	<node pkg="dnn_attitude_estimation" name="camera_regression_mcdropout_inference" type="camera_regression_mcdropout_inference.py" output="screen">
		<remap from="/color_image" to="/decompressed_image"/>
	</node>
</launch>
